tox.ini

```
@@ -10,11 +10,21 @@ setenv =
 passenv = HOME
 
 [testenv:unit]
+setenv =
+  PYTHONPATH = {toxinidir}/lib/:{toxinidir}:{toxinidir}/src/::{toxinidir}/lib/
 commands = 
     coverage run -m unittest discover -s {toxinidir}/tests/unit -v
     coverage report --omit tests/*,mod/*,.tox/*
     coverage html --omit tests/*,mod/*,.tox/*
-deps = -r{toxinidir}/tests/unit/requirements.txt
+deps = 
+    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/tests/unit/requirements.txt
+
+[testenv:integration]
+commands = 
+    pytest {toxinidir}/tests/integration
+deps = 
+    -r{toxinidir}/tests/integration/requirements.txt
 
 [testenv:lint]
 commands = flake8 src/
```

```suggestion
commands = flake8 srctests/
```

Do you mind also linting the code?